### PR TITLE
fix argo cd related issues with playground setup

### PR DESCRIPTION
**Pull request by hpgrahsl**

### CODE REVIEW

**Comment by cmoulliard** [comment]
I did a end to end test and I was able locally to:
- Scaffold a new quarkus app and create some argocd resources
- Consult the argocd history
Screenshot 2024-02-07 at 15 19 43

**Review by cmoulliard — APPROVED** [review]

**Comment by cmoulliard** [comment]
Can you fix the following error please ? @hpgrahsl 
```
yarn run v1.22.21
$ tsc
packages/app/src/components/catalog/EntityPage.tsx:188:8 - error TS2746: This JSX tag's 'children' prop expects a single child of type 'Element', but multiple children were provided.

188       <EntityLayout.Route path="/cd" title="CD">
           ~~~~~~~~~~~~~~~~~~


Found 1 error in packages/app/src/components/catalog/EntityPage.tsx:188
```

**Comment by hpgrahsl** [comment]
should be fine now @cmoulliard

File: packages/backend/src/plugins/argocd.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { createRouter } from '@roadiehq/backstage-plugin-argo-cd-backend';
```
Comment (cmoulliard) [inline packages/backend/src/plugins/argocd.ts:1]: Question: Where did you find this information ? @hpgrahsl

**Pull request merged**